src/url.rs

```diff
@@ -29,35 +85,46 @@ impl Scanner for UrlScanner {
 }
 
 impl UrlScanner {
+    // For URL searching starting before the `://` separator, the `has_scheme` parameter should be
+    // true because the URL will have a scheme for sure. If seraching before the `.` separator, it
```

Comment:
seraching -> searching

Reply:
Thanks! I should really fix my Vim spell checker.